addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -922,6 +922,40 @@ QUnit.test('Attachments that have been unlinked from server should be visually u
     );
 });
 
+QUnit.test('the chatter should be empty when creating a new record after having displayed a chatter for an existing record', async function (assert) {
```

Comment:
Name of test doesn't match what it actually does.
More like asserts there's only a 'Creating a new record...' message after click on "create" in control panel.

Suggestion: _"Chatter just contains 'Creating a new record' message during creation of a new record"_

Reply:
Should still keep the part about "after having displayed a chatter for an existing record" because that was the issue.